R/02-pilot-locality-regions.R

```diff
@@ -0,0 +1,156 @@
+# Copyright 2025 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script loads aggregated csv data files containing spatial data for 
+# municipality of interest in BC (loc). 
+# It plots the locations on top of a map of B.C. 
+
+# ------------------------------------------------------------------------
+# Script: 02-plot-localities.R
+
+# Description: plot localities on a map of BC
+
+# Requirements:
+#   - Requires R packages: `tidyverse`, `glue`, `janitor`, `sf`. `rmapshaper`, `rnaturalearth`
+#   - Depends on `settings.R` for paths and constants.
+#   - Requires input DA/DB shapefiles and the crosswalk CSV file.
+#   - Requires read/write access to relevant data folders.
+
+# Side Effects/Outputs:
+#   - Makes a map of localities we used
+# ------------------------------------------------------------------------
+
+library(tidyverse)
+library(glue)
+library(janitor)
+library(sf)
+library(rmapshaper)    # simplify geometries
+library(rnaturalearth) # get BC outline
```

Comment:
from Steph H: "Noticed you are pulling bc admin boundary from rnaturalearth, just a reminder that BC manages these boundaries as well and data is available in the catalogue (and so from bcdata or bcmaps), including a regular and highres BC boundary."

Reply:
good call. Updating